images/arch/extra-packages

```diff
@@ -0,0 +1,27 @@
+bash-completion
```

Comment:
If you were to include the 'base' group, then most likely some packages can be removed from this list.

Reply:
Yup, I've removed the packages that belong to base and base-devel.